frontend/src/components/common/table/table.tsx

```diff
@@ -21,42 +22,45 @@ const Table = <Data extends Record<string, unknown>>({
     tableInstance;
 
   return (
```

Comment:
also, add a new optional prop "placeholder" with a default "No data to display" value